cumulus/parachain-template/node/Cargo.toml

```diff
@@ -62,6 +62,7 @@ xcm = { package = "staging-xcm", path = "../../../polkadot/xcm", default-feature
 cumulus-client-cli = { path = "../../client/cli" }
 cumulus-client-collator = { path = "../../client/collator" }
 cumulus-client-consensus-aura = { path = "../../client/consensus/aura" }
+cumulus-primitives-pov-reclaim = { path = "../../primitives/pov-reclaim" }
```

Comment:
We probably should re-export the host functions from `cumulus-client-service` for easier consumption.